all_repos/github_api.py

```diff
@@ -47,6 +47,18 @@ def get_all(url: str, **kwargs: Any) -> List[Dict[str, Any]]:
     return ret
 
 
+def filter_repos_from_settings(
+    repos: List[Dict[str, Any]], settings: Any,
```

Comment:
this makes the call not typesafe, which is why it wasn't moved in the first place

there's no reason to have both `filter_repos` and `filter_repos_from_settings` as well

I would just drop this commit

Reply:
removed